.github/renovate.json

```diff
@@ -37,6 +37,7 @@
                 "mirror/ncurses",
                 "PCRE2Project/pcre2",
                 "plougher/squashfs-tools",
+                "SELinuxProject/selinux",
```

Comment:
let's use `https://github.com/SELinuxProject/selinux.git` here

Reply:
ok, currently on the tcl stuff, will fix and then return here (if not busy with other stuff)